lib/ansible/modules/cloud/vmware/vmware_guest_boot_facts.py

```diff
@@ -20,10 +20,10 @@
 module: vmware_guest_boot_facts
 short_description: Gather facts about boot options for the given virtual machine
 description:
-    - This module can be used to gather facts aboyt boot options for the given virtual machine.
+    - This module can be used to gather facts about boot options for the given virtual machine.
```

Comment:
```suggestion
    - Gather facts about boot options for the given virtual machine.
```
If it can't be used for something else

Reply:
OK.